### DISCUSSION TITLE
ЛР 3 для КТ. Ошибка в методичке

### DISCUSSION THREAD

**Discussion by RaidMachine** [MPSU/exAPS#30] [Ошибки]
Здравствуйте, в методичке к лабораторной работе по регистровому файлу и памяти ошибка в коде. У переменной new_data один разряд, а надо 4 для записи в него строк из файла.
![image](https://user-images.githubusercontent.com/61085736/196462849-efe701e5-3adc-4122-a8d6-63242aec80a3.png)

**Comment by gr33nka** [comment]
Круть! Спасибо! Исправим

@HepoH3 организуйте, пожалуйста, дополнительные пол балла к лабе или курсу внимательному студенту. Помнится мы планировали такую практику вводить

**Comment by HepoH3** [comment]
Нет проблем, студенту только нужно раскрыть свое инкогнито )

**Reply by gr33nka** [reply]
Ты потри потом это сообщенице в целях цифровой гигиены)

**Reply by HepoH3** [reply]
Готово
